src/bitcoin-cli.cpp

```diff
@@ -884,6 +884,33 @@ static void GetWalletBalances(UniValue& result)
     result.pushKV("balances", balances);
 }
 
+/**
+ * GetProgressBar get a progress bar. The increment of the progress bar is 5%.
+ *
+ * @param progress  The fraction of the progress bar to be filled. Min: 0.0, max: 1.0.
+ * @returns a string representation of the progress bar.
+ */
+static std::string GetProgressBar(const double progress)
+{
+    if (progress < 0 || progress > 1) return "";
+
+    std::string progress_bar = "|";
+    static constexpr double INCREMENT{0.05};
+    static const std::string COMPLETE_BAR{"\u2588"};
+    static const std::string INCOMPLETE_BAR{"\u2591"};
+
+    for (int i = 0; i < progress / INCREMENT; i++) {
+        progress_bar += COMPLETE_BAR;
+    }
+
+    for (int i = 0; i < (1 - progress) / INCREMENT; i++) {
+        progress_bar += INCOMPLETE_BAR;
+    }
+
+    progress_bar += "|";
```

Comment:
```suggestion
    std::string progress_bar = "";
    static constexpr double INCREMENT{0.05};
    static const std::string COMPLETE_BAR{"\u2588"};
    static const std::string INCOMPLETE_BAR{"\u2591"};

    for (int i = 0; i < progress / INCREMENT; i++) {
        progress_bar += COMPLETE_BAR;
    }

    for (int i = 0; i < (1 - progress) / INCREMENT; i++) {
        progress_bar += INCOMPLETE_BAR;
    }

    progress_bar += "";
```

![image](https://user-images.githubusercontent.com/13405205/126921029-27786d8f-f327-4cc1-b11a-6cf51347d7c6.png)

It looks okay even without `|`

Reply:
Done. Thanks 👍